vrep_interface_tests/synchronous_test/synch_test.py

```diff
@@ -0,0 +1,67 @@
+#!/bin/python3
+"""
+(C) Copyright 2023 DQ Robotics Developers
+This file is part of DQ Robotics.
+    DQ Robotics is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Lesser General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+    DQ Robotics is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU Lesser General Public License for more details.
+    You should have received a copy of the GNU Lesser General Public License
+    along with DQ Robotics.  If not, see <http://www.gnu.org/licenses/>.
+
+Contributors:
+- Juan Jose Quiroz Omana (juanjqo@g.ecc.u-tokyo.ac.jp)
+
+Instructions:
+    Prerequisites:
+    - dqrobotics
+    - dqrobotics-interface-vrep
+
+1) Open the CoppeliaSim scene synch_test.ttt
+2) Run and enjoy!
+"""
+
+from dqrobotics.interfaces.vrep import DQ_VrepInterface
+import time
+
+
+def main() -> None:
+    vi = DQ_VrepInterface()
+    try:
+        vi.connect("127.0.0.1", 19997, 100, 10)
```

Comment:
@juanjqo Please test for a valid connection and raise an Exception if it does not connect.

Reply:
@mmmarinho I added the test, as suggested.